src/main/java/seedu/clinkedin/ui/PersonCard.java

```diff
@@ -123,7 +123,9 @@ public PersonCard(Person person, int displayedIndex, MainWindow mainWindow) {
         rating.setPadding(new Insets(1));
 
         CornerRadii corn = new CornerRadii(2.5);
-        if (ratingNumber <= 3 && ratingNumber >= 1) {
+        if (ratingNumber == 0) {
+            // do not show anything
```

Comment:
Might be bad code practice to just have empty clauses. Are you able to somehow refactor this/remove this clause while making the logic intact?

Reply:
ok